test/scenarios/el92_prev-minor_upgrade-failing.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
This might not work, 4.13 probably will not recover from a rollback (or just a happy by-product)

Reply:
We could wait to add this until 4.15 opens.